care_o_bot_desktop/package.xml

```diff
@@ -20,7 +20,7 @@
   <exec_depend>language-pack-en</exec_depend>
   <exec_depend>language-pack-de</exec_depend>
   <exec_depend>openssh-server</exec_depend>
-  <exec_depend>chromium-browser</exec_depend>
+  <!--exec_depend>chromium-browser</exec_depend-->
```

Comment:
Yes, they are. At least it seems like it.
During release, it tells me that the fedora rosdep key is not found...it also offers me to skip, but has a very thorough warning saying only to do so in case I'm __absolutely__ sure that the key is not __available__ in fedora...this is why I did __not__ skip it :wink: